Sources/Testing/Traits/Bug.swift

```diff
@@ -59,42 +59,42 @@ extension Bug: TestTrait, SuiteTrait {
 }
 
 extension Trait where Self == Bug {
-  /// Construct a bug to track with a test.
+  /// Constructs a bug to track with a test.
```

Comment:
Now that I think about it… does Swift normally use "construct" here or "initialize" or…? I ask because Swift doesn't have _constructors_ like C++ or some other languages.

Reply:
Not directly relevant to the PR, but we should make sure the guidelines cover factory functions like these.